bigquery_etl/glam/templates/extract_probe_counts_v1.sql

```diff
@@ -1,48 +1,42 @@
 {{ header }}
--- TODO: Remove deduping when dupes are fixed.
-WITH deduped AS (
-    SELECT
-        *,
-        ROW_NUMBER() OVER(
-            PARTITION BY
-                ping_type,
-                os,
-                app_version,
-                app_build_id,
-                channel,
-                metric,
-                metric_type,
-                key,
-                client_agg_type,
-                agg_type
-            ORDER BY
-                total_users DESC
-        ) AS rank
-    FROM
-        `{{ dataset }}.{{ prefix }}__view_probe_counts_v1`
-    WHERE
-        channel IS NOT NULL
-        AND app_version IS NOT NULL
-        AND total_users >= 100
-)
+
+CREATE TEMP FUNCTION udf_js_flatten(histogram ARRAY<STRUCT<key STRING, value FLOAT64>>)
+RETURNS STRING
+LANGUAGE js
+AS
+  '''
+    let obj = {};
+    histogram.map(function(r) {
+        obj[r.key] = parseFloat(r.value.toFixed(4));
+    });
+    return JSON.stringify(obj);
+''';
 
 SELECT
     channel,
     app_version as version,
-    COALESCE(ping_type, "*") as ping_type,
-    COALESCE(os, "*") AS os,
-    COALESCE(app_build_id, "*") AS build_id,
+    ping_type,
+    os,
+    app_build_id as build_id,
     metric,
     metric_type,
     -- BigQuery has some null unicode characters which Postgresql doesn't like,
     -- so we remove those here. Also limit string length to 200 to match column
     -- length.
     SUBSTR(REPLACE(key, r"\x00", ""), 0, 200) AS metric_key,
     client_agg_type,
-    agg_type,
-    total_users,
-    TO_JSON_STRING(aggregates) AS data
+    MAX(total_users) as total_users,
```

Comment:
FYI: I had `total_users` in the `GROUP BY` but there was a bug for desktop where these values were different, so I moved it here. This is just a temporary fix until desktop ETL is fixed. You probably don't need this.

Reply:
Yea. Thanks for pointing this out. The fix is coming here: https://github.com/mozilla/bigquery-etl/pull/1004